### PR TITLE
Jimmy

### PR DESCRIPTION
**Pull request by yitaoli97**
please review this screen

### CODE REVIEW

File: src/main/controllers/AttendeeScreenController.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    /**
+     * Constructor of an AttendeeScreenController.
+     */
+    public AttendeeScreenController() {}
+
+    /**
+     * Checks valid input from attendee and tells ProgramController what screen to go next.
+     */
+    public String run() {
+        AttendeeScreen attendeeScreen = new AttendeeScreen();
```
Comment (bravetheheat) [inline src/main/controllers/AttendeeScreenController.java]: You would likely want to make this a class variable

File: src/main/controllers/AttendeeScreenController.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        String[] options = {"all events", "registered events", "contacts", "messages"};
```
Comment (bravetheheat) [inline src/main/controllers/AttendeeScreenController.java]: I would do a numeric selection

File: src/main/controllers/AttendeeScreenController.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        List<String> prompts = (Arrays.asList(options));
+        attendeeScreen.prompt();
+        Scanner sc = new Scanner(System.in);
```
Comment (bravetheheat) [inline src/main/controllers/AttendeeScreenController.java:34]: Make this a class variable

**Review by bravetheheat — COMMENTED** [review]
The idea is there, so good work :)

**Comment by yitaoli97** [comment]
implemented suggested changes

**Comment by yitaoli97** [comment]
changed the logic of switching screens in ProgramController
- added abstract ScreenController class
- starts from LoginScreenController
- run currentScreenController, then change currentScreeenController to the following ScreenController based on the output of currentScreenController

**Review by bravetheheat — APPROVED** [review]

**Pull request merged**